src/relayFeeCalculator/chain-queries/arbitrum.ts

```diff
@@ -9,22 +9,12 @@ export class ArbitrumQueries extends QueryBase {
     provider: providers.Provider,
     symbolMapping = TOKEN_SYMBOLS_MAP,
     spokePoolAddress = getDeployedAddress("SpokePool", CHAIN_IDs.ARBITRUM),
-    usdcAddress = TOKEN_SYMBOLS_MAP.USDC.addresses[CHAIN_IDs.ARBITRUM],
     simulatedRelayerAddress = DEFAULT_SIMULATED_RELAYER_ADDRESS,
     coingeckoProApiKey?: string,
     logger: Logger = DEFAULT_LOGGER,
     gasMarkup = 0
   ) {
-    super(
-      provider,
-      symbolMapping,
-      spokePoolAddress,
-      usdcAddress,
-      simulatedRelayerAddress,
-      gasMarkup,
-      logger,
-      coingeckoProApiKey
-    );
+    super(provider, symbolMapping, spokePoolAddress, simulatedRelayerAddress, gasMarkup, logger, coingeckoProApiKey);
```

Comment:
A common paradigm in this PR - we can't really specify a specific address anymore since we need this context to be passed into the `handleAcrossMessage` function